src/wallet/wallet.cpp

```diff
@@ -3298,7 +3298,7 @@ bool CWallet::InitLoadWallet()
             CKey key;
             key.MakeNewKey(true);
             if (!walletInstance->SetHDMasterKey(key))
-                throw std::runtime_error("CWallet::GenerateNewKey(): Storing master key failed");
+                throw std::runtime_error("CWallet::InitLoadWallet(): Storing master key failed");
```

Comment:
It's kinda dead code now, but we should keep this check for further changes in `SetHDMasterKey()`.